frontend/public/components/dashboard/dashboards-page/cluster-dashboard/utilization-card.tsx

```diff
@@ -168,42 +168,67 @@ export const PrometheusUtilizationItem = withDashboardResources<PrometheusUtiliz
     utilizationQuery,
     totalQuery,
     duration,
+    adjustDuration = (start: number) => start,
```

Comment:
Its better to not assign a default value, because for every rerender the function will be a new one which will make the use of `useMemo` below useless.

Reply:
All my comments for `PrometheusUtilizationItem` apply to `PrometheusMultilineUtilizationItem` too